pkg/controllers/managementuser/secret/project_scoped_secrets.go

```diff
@@ -32,22 +34,29 @@ const (
 	projectScopedSecretLabel = "management.cattle.io/project-scoped-secret"
 	pssCopyAnnotation        = "management.cattle.io/project-scoped-secret-copy"
 )
+const (
+	normanLabel      = "cattle.io/creator"
```

Comment:
nit: IMHO doesn't read well, maybe make a bit more concrete?
```suggestion
	normanCreatorLabel      = "cattle.io/creator"
```